### PR TITLE
[feature][Install] 6597 add worker group when upgrading from version 1.x to 2.0

### LINKED ISSUE #6597: [Feature][Install] Upgrade dolphinscheduler version 1. X to version 2.0
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

we want upgrade dolphinscheduler from version 1.x to 2.0

1.  some db table structure changes.
2.  some definition data need to be translated

### Use case

_No response_

### Related issues

_No response_

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by lenboo**
close #6597 add worker group when upgrading from version 1.x to 2.0

### CODE REVIEW

**Review by brave-lee — DISMISSED** [review]
+1

**Review by GaoTianDuo — CHANGES_REQUESTED** [review]
"id serial int NOT NULL " please don't add int

File: dolphinscheduler-dao/src/main/resources/sql/upgrade/2.0.0_schema/postgresql/dolphinscheduler_ddl.sql [hunk 1]
```diff
@@ -210,7 +210,7 @@ BEGIN
 	)';
 
 	EXECUTE 'CREATE TABLE IF NOT EXISTS '|| quote_ident(v_schema) ||'."t_ds_task_definition" (
-	  id int NOT NULL  ,
+	  id serial int NOT NULL  ,
```
Comment (GaoTianDuo) [inline dolphinscheduler-dao/src/main/resources/sql/upgrade/2.0.0_schema/postgresql/dolphinscheduler_ddl.sql]: id serial not null or id serial

File: dolphinscheduler-dao/src/main/resources/sql/upgrade/2.0.0_schema/postgresql/dolphinscheduler_ddl.sql [hunk 2]
```diff
@@ -298,6 +298,17 @@ BEGIN
 	  update_time timestamp DEFAULT NULL ,
 	  PRIMARY KEY (id)
 	)';
+
+	EXECUTE 'CREATE TABLE IF NOT EXISTS '|| quote_ident(v_schema) ||'."t_ds_worker_group" (
+      id serial int(11) NOT NULL,
```
Comment (GaoTianDuo) [inline dolphinscheduler-dao/src/main/resources/sql/upgrade/2.0.0_schema/postgresql/dolphinscheduler_ddl.sql]: in postgres please use int or bigint  exp: id  int not null

**Review by GaoTianDuo — CHANGES_REQUESTED** [review]

File: dolphinscheduler-dao/src/main/resources/sql/upgrade/2.0.0_schema/postgresql/dolphinscheduler_ddl.sql (hunk 1 above)
Comment (lenboo) [reply dolphinscheduler-dao/src/main/resources/sql/upgrade/2.0.0_schema/postgresql/dolphinscheduler_ddl.sql]: > id serial not null or id serial

thanks for reminding!

**Review by brave-lee — APPROVED** [review]
+1

**Pull request merged**